recipes/hyperspy_gui_ipywidgets/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "hyperspy-gui-ipywidgets" %}
+{% set filename = "hyperspy_gui_ipywidgets" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "31803a08d74493b697714ca759a217568fda08a33dad3d39dfd9bf3bf0923fbc" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ filename }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - link-traits
+    - hyperspy >=1.3
+    - ipywidgets >=6.0
+
+test:
+  imports:
+    - hyperspy_gui_ipywidgets
+
+  commands:
+    - export MPLBACKEND="agg"  # [unix]
+    - py.test --pyargs hyperspy_gui_ipywidgets  # [unix]
```

Comment:
At present the tests don't run in Windows. This is because hyperspy imports matplotlib which defaults to the Qt backend that requires a screen. In unix it is possible to change the default matplotlib backend by just setting the MPLBACKEND environement variable but I don't think this is possible in Windows. I think that the matplotlib recipe workarounds this issue by patching matplotlib.